package.json

```diff
@@ -8,46 +8,33 @@
   "license": "Mozilla Public License 2.0",
   "scripts": {
     "analyze": "npx source-map-explorer 'build/static/js/*.js'",
-    "build": "react-scripts build",
-    "serve-build": "concurrently \"npm:bff-api:serve-build\" \"node scripts/serveBuild\"",
-    "serve-dev": "concurrently \"npm:bff-api:watch\" \"npm:start\"",
     "release:bug": "sh scripts/release.sh --bug",
     "release:major": "sh scripts/release.sh --major",
     "release": "sh scripts/release.sh --minor",
-    "start": "cross-env REACT_APP_ENV=development react-scripts start",
-    "test": "react-scripts test src/client",
+    "start": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "vitest src/client",
```

Comment:
Krijgen we ook weer een commando om zowel server als client te starten?

Reply:
Oh, ja als je dat wilt prima.